drivers/all_neighbors_system_dashboard/config/routes.rb

```diff
@@ -0,0 +1,7 @@
+BostonHmis::Application.routes.draw do
+  namespace :all_neighbors_system_dashboard do
+    namespace :warehouse_reports do
+      resources :all_neighbors_system_dashboards
```

Comment:
maybe this should just be:
```suggestion
      resources :reports
```